### PR TITLE
[HOW-TO] Create a Manual Model from function and derivatives

### PR DESCRIPTION
**Pull request by abelsiqueira**
Needs SolverBenchmark.jl to be updated so BenchmarkTools.jl isn't restricted.

### CODE REVIEW

**Comment by dpo** [comment]
Which SB PR is that>

**Comment by abelsiqueira** [comment]
There's a PR by CompatHelper, but it's probably better to check which versions were lost due to the inactivity of the action.

**Comment by abelsiqueira** [comment]
Now failing because LinearOperators was updated to version 2 and the tutorial hasn't.

File: _literate/how-to/create-a-manual-model.jl [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  @views δ = (sigmoid.(β[1] .+ X * β[2:end]) - y) / n
+  out[1] = sum(δ) + 1e-4β[1]
+  @views out[2:end] .= X' * δ + 1e-4 * β[2:end]
+  return out
+end
+
+nlp = NLPModel(
+  zeros(p + 1),
+  β -> myfun(β, X, y),
+  grad=(out, β) -> mygrad(out, β, X, y),
```
Comment (tmigot) [inline _literate/how-to/create-a-manual-model.jl:51]: The tutorial looks great, thanks. Should we precise that the given function must be in-place?

**Review by tmigot — APPROVED** [review]

File: _literate/how-to/create-a-manual-model.jl (hunk 1 above)
Comment (abelsiqueira) [reply _literate/how-to/create-a-manual-model.jl:51]: Done

**Comment by abelsiqueira** [comment]
Thanks for the review!

**Pull request merged**